### DISCUSSION TITLE
Create inside of update

### DISCUSSION THREAD

**Discussion by robertklein** [prisma/prisma#4518] [Q&A] [answered]
I want to add 2 new categoryTrans inside of update category. I tried something like this:

```js
ctx.prisma.category.update({
  where: {
    id: 64,
  },
  data: {
    is_active: 1,
    categoryTrans: {
      create: [
        { name: 'some de name', language: { connect: { id: 'de' }} },
        { name: 'some en name', language: { connect: { id: 'en' }} }
      ]
    }
  },
})
```
but I got an error:
```js
Error for binding '2': AssertionError(\"[Query Graph] Expected a valid parent ID to be present for a nested connect on a one-to-many relation.\")
Query interpretation error. Error for binding '2': AssertionError(\"[Query Graph] Expected a valid parent ID to be present for a nested connect on a one-to-many relation.\")
```

Any help please?

Models:
```js
model language {
  id         String       @id
  name       String
  category_t category_t[]
}

model category {
  id             Int          @id @default(autoincrement())
  parent_id      Int?
  level          Int
  sort           Int?
  is_active      Int          @default(1)
  created_at     DateTime     @default(now())
  updated_at     DateTime     @default(now()) @updatedAt
  deleted_at     DateTime?
  subCategories  category[]   @relation("categoryTocategory")
  parentCategory category?    @relation("categoryTocategory", fields: [parent_id], references: [id])
  categoryTrans  category_t[]
}

model category_t {
  id          Int      @id @default(autoincrement())
  ref_id      Int
  name        String
  language_id String
  category    category @relation(fields: [ref_id], references: [id])
  language    language @relation(fields: [language_id], references: [id])

  @@unique([language_id, ref_id], name: "category_t_language_id_ref_id_unique_constraint")
}
```

**Comment by ryands17** [comment, marked as answer]
Hey @robertklein 👋 

I tried the same query by running the above query on this schema and it works fine. I created the category and required languages first and then ran the `update` query that you have provided above.

Could you check if the `language` that you are using `connect` for exist in the database? Also I'm uing version 2.12.1 of `@prisma/client` and `@prisma/cli`.

**Reply by robertklein** [reply]
thanks a lot @ryands17 , 'de' was missing in langs. Shame on me!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
